todo_files/tests/integration_test.py

```diff
@@ -44,9 +44,3 @@ def mock_get_cards(url):
         response.json.return_value = sample_trello_lists_response
```

Comment:
It's worth noting that these checks will only pass for your specific list ids, so your tests aren't configurable using the environment variables as this stands